packages/react-components/src/components/CameraButton.tsx

```diff
@@ -149,34 +159,54 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     strings.tooltipDisabledContent = strings.tooltipVideoLoadingContent;
   }
 
+  const cameraOn = props.checked;
+
+  const toggleAnnouncerString = useCallback(
+    (isCameraOn: boolean) => {
+      setAnnouncerString(
+        announcerString === strings.cameraActionTurnedOffAnnouncement || !isCameraOn
```

Comment:
Non blocking question as I tested on android and seems to be working correctly (except for when focusing on a local video tile then back to the button):
I'm confused as to why announcerString is being compared to strings.cameraActionTurnedOffAnnouncement.
Would it not always set announcerString to the off string if announcerString is already the off string? It would not reach the !isCameraOn condition.